WooCommerce/Classes/POS/Presentation/Item Selector/ItemList.swift

```diff
@@ -48,20 +48,20 @@ struct ItemList<HeaderView: View>: View {
     }
 
     @ViewBuilder var footerRows: some View {
-        VStack {
-            switch state {
-            case .loading:
+        switch state {
+        case .loading:
+            ForEach(0..<8) { _ in
                 GhostItemCardView()
```

Comment:
nit;

Having a smoother transition from the ghost cell to an actual cell would be nice.  But it will likely require `.matchedGeometryEffect` + `.transition` as in `subtotalFieldView`.

Reply:
I agree, but didn't want to dive in to that until I move over to Observation – at the moment the item list state change is quite abrupt from SwiftUI's point of view, and it may be easier to animate when that's more granular.